### PR TITLE
#198 controls improvements

### PR DESCRIPTION
**Pull request by m-m-mic**
Improves the design and functionalities of all three controls (switch, checkbox & radio) based on UI review feedback.

### CODE REVIEW

File: mwui-stencil/src/components/mw-switch/mw-switch.tsx [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 @Component({
   tag: "mw-switch",
-  styleUrl: "mw-switch.css",
-  shadow: true,
+  styleUrl: "mw-switch.scss",
+  shadow: false,
 })
 export class MwSwitch {
-  private checkbox: HTMLInputElement;
+  private switchId = `switch-input-${switchIds++}`;
```
[the remark this answers is not in the document]
Comment (m-m-mic) [reply mwui-stencil/src/components/mw-switch/mw-switch.tsx:11]: I wanted the ids for all three control types to work the same (switch, radio, checkbox). We can use the name but then we have to verify that the name is unique as two switches with the same id causes unintended behavior. (e.g. if you press on the label of one of the switches, both will appear as :active).

File: mwui-stencil/src/components/mw-switch/mw-switch.tsx [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
+import { Component, Event, EventEmitter, Prop, h, Host } from "@stencil/core";
+import { Position } from "../../shared/models/enums/position.enum";
+let switchIds = 0;
 
 @Component({
   tag: "mw-switch",
-  styleUrl: "mw-switch.css",
-  shadow: true,
+  styleUrl: "mw-switch.scss",
+  shadow: false,
```
[the remark this answers is not in the document]
Comment (m-m-mic) [reply mwui-stencil/src/components/mw-switch/mw-switch.tsx:8]: Good point. In this case I did it for two reasons: 
- None of the controls really benefit from having a shadow-dom while disabling it made it easier for me to implement label position, active/focused state behavior, and intended cursor behavior without needing to add extra divs/spans to the component
- As far as I'm aware we can't use the shadow-dom for radio buttons due to how the radio group was implemented and while refactoring the controls I wanted to make them as structurally and functionally similar as possible (should make changing them down the line a little less frustrating)

File: mwui-stencil/src/components/mw-switch/mw-switch.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Fuasmattn) [reply mwui-stencil/src/components/mw-switch/mw-switch.tsx:12]: so you wanted to have unique ids for the switches internally? 
then I would recommend using a uuid (not yet a dependency).

[the remark this answers is not in the document]
Comment (m-m-mic) [reply mwui-stencil/src/components/mw-switch/mw-switch.tsx]: Ok, I've replaced the ids for all three controls with uuidv4. Is that what you had in mind?

**Pull request merged**